clients/python/test/e2e/test_solvers_api.py

```diff
@@ -63,6 +51,17 @@ def test_get_jobs(configuration: osparc.Configuration):
         for elm in tmp_iter:
             assert isinstance(elm, osparc.Job)
 
+        async def return_iter_content(
```

Comment:
I get this but perhaps you can write a separate async test for clarity? 
both tests should like the same except that in one case is ``asyng test_`` and the other is not

Reply:
I have deleted the async part of the iterator for now. Reason: since I use the links I get from the first call to the api directly, the iterator now needs to hold its own http client. Hence, to have a sync and an async version it makes sense to have two different types of iterators: One holding a sync client and one holding an async client. Until someone tells me we need the async one I will stick with the sync one.